vistas/vendedores/actualizar.php

```diff
@@ -0,0 +1,15 @@
+<main class="contenedor seccion">
+  <h1>Actualizar Vendedor(a)</h1>
+
+  <?php foreach ($errores as $error) : ?>
+    <div class="alerta error">
+      <?php echo $error; ?>
+    </div>
+  <?php endforeach ?>
+  <form class="formulario" method="POST">
+    <?php include __DIR__ . "/formulario.php" ?>
+    <input type="submit" value="Actualizar Vendedor(a)" class="boton boton-verde">
```

Comment:
```suggestion
    <input type="submit" value="Actualizar Vendedor" class="boton boton-verde">
```
Realiza este cambio por favor